File: app/src/main/java/com/idemia/skycy/models/SKYResponse.java
package com.idemia.skycy.models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class SKYResponse {
    @SerializedName("title")
    private String title;
    @SerializedName("overview")
    private String overview;
    @SerializedName("duration")
    private String duration;
    @SerializedName("release_year")
    private String release_year;
    @SerializedName("cover_url")
    private String cover_url;
    @SerializedName("id")
    private String id;
    @SerializedName("backdrops_url")
    private List<String> backdrops_url;

    public String getTitle() {
        return title;
    }

    public String getOverview() {
        return overview;
    }

    public String getDuration() {
        return duration;
    }

    public String getRelease_year() {
        return release_year;
    }

    public String getCover_url() {
        return cover_url;
    }

    public String getId() {
        return id;
    }

    public List<String> getBackdrops_url() {
        return backdrops_url;
    }
}

File: app/src/main/java/com/idemia/skycy/models/SKYResponse.kt
package com.idemia.skycy.models

import com.google.gson.annotations.SerializedName

class SKYResponse {
    @SerializedName("title")
    val title: String? = null

    @SerializedName("overview")
    val overview: String? = null

    @SerializedName("duration")
    val duration: String? = null

    @SerializedName("release_year")
    val release_year: String? = null

    @SerializedName("cover_url")
    val cover_url: String? = null

    @SerializedName("id")
    val id: String? = null

    @SerializedName("backdrops_url")
    val backdrops_url: List<String>? = null

}